pkg/route/api/allocator/allocator.go

```diff
@@ -0,0 +1,30 @@
+package allocator
```

Comment:
Still muddling through ... But I am going to close this PR out and create a new one.